lib_test/test_common.ml

```diff
@@ -110,7 +110,14 @@ let create: (module Irmin.S_MAKER) -> [`String | `Json] -> (module Irmin.S) =
     in
     let module S = Irmin.Basic(B)(C) in (module S)
 
-type kind = [`Mem | `Fs | `Git | `Http of kind]
+type kind = [`Mem
+	    | `Fs
+	    | `Git
+	    | `Link_FS_Chunck_Krypto
+	    | `Link_MEM_Chunck_Krypto
+	    | `Link_FS_Chunck
+	    | `Link_MEM_Chunck
+	    | `Http of kind]
```

Comment:
Also I think that type would look better with something like:

```ocaml
type kind = [ 
  | `Mem
  | `Git
  | `Http of kind
  | `Krypto of kind
  | `Chunk of kind
  | `Link of kind
```

I guess all the combinations are possible although some of them don't really make sense.